package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "local-home-virual-environment",
+  "name": "local-home-virtual-environment",
```

Comment:
what about `local-home-testing` ?

Reply:
Thanks, was looking for a better name for this.  Will change

Reply:
Done